spec/features/admin/tag_rules_spec.rb

```diff
@@ -189,25 +195,25 @@
       expect(default_fsm_tag_rule.preferred_matched_shipping_methods_visibility).to eq "hidden"
 
       # FilterShippingMethods rule
-      expect(fsm_tag_rule.reload.priority).to eq 1
+      expect(fsm_tag_rule.reload.priority).to eq 4
```

Comment:
Hi @bingxie,

Thanks for this work!

I have just found one problem with this commit. The values that you have changed here were correct before. The values are as they are for a reason, so if the spec is failing, it means that the page is broken.

I noticed that you have done a similar thing before, with this commit: d59dea29ab8967fa8bb19eda77ba460e87d4860a, where you commented out the lines that were making the spec fail. I had to add these lines back in 9dc1294ec527e239ef0c9b75dfb05a2f00f856aa, because they were testing important page functionality.

You need to be careful that you are not just changing spec code so that you get a green build. While this is sometimes the right thing to do because the spec is obsolete, in most cases it will be because the page needs fixing.

If you are unsure about a broken spec, feel free to ask me for help any time. This particular issue was tricky to track down, but I have resolved it for now in 9d61d538e9131e08f916305bffe8c4974639cd6e.

Thanks again! 😄